tools/vscode_sql_extraction/src/sql_extraction_runner.ts

```diff
@@ -54,7 +55,26 @@ export class SqlExtractionRunner {
       process.stdout!.on('data', data => {
         json += data;
       });
-      // todo: track progress
+      process.stderr!.on('data', data => {
+        errMsg += data;
+        let index = errMsg.indexOf('\n');
+        while (index >= 0) {
```

Comment:
Seems easier to handle multiple '\n' by something like this:

```
if (index >= 0) {
  var lines = errMsg.split('\n');

  // assign last line to errMsg
  errMsg = lines[lines.length - 1];

  // display other lines
  for (var i = 0; i < line.length - 1; ++i)  {
    progress.report({
              message: line[i],
              increment: isNaN(percent) ? undefined : percent,
     });
  }
}
```

Reply:
Great suggestion, thanks!